modules/solid_mechanics/src/auxkernels/MaterialTensorCalculator.C

```diff
@@ -88,54 +89,58 @@ MaterialTensorCalculator::getTensorQuantity(const SymmTensor & tensor,
       break;
 
     case 3:
-      value = MaterialTensorCalculatorTools::hydrostatic(tensor);
+      value = MaterialTensorCalculatorTools::effectiveStrain(tensor);
       break;
 
     case 4:
-      value = MaterialTensorCalculatorTools::directionValueTensor(tensor, _direction);
+      value = MaterialTensorCalculatorTools::hydrostatic(tensor);
       break;
 
     case 5:
-      value = MaterialTensorCalculatorTools::hoopStress(tensor, _p1, _p2, curr_point, direction);
+      value = MaterialTensorCalculatorTools::directionValueTensor(tensor, _direction);
       break;
 
     case 6:
-      value = MaterialTensorCalculatorTools::radialStress(tensor, _p1, _p2, curr_point, direction);
+      value = MaterialTensorCalculatorTools::hoopStress(tensor, _p1, _p2, curr_point, direction);
       break;
 
     case 7:
-      value = MaterialTensorCalculatorTools::axialStress(tensor, _p1, _p2, direction);
+      value = MaterialTensorCalculatorTools::radialStress(tensor, _p1, _p2, curr_point, direction);
       break;
 
     case 8:
-      value = MaterialTensorCalculatorTools::maxPrinciple(tensor, direction);
+      value = MaterialTensorCalculatorTools::axialStress(tensor, _p1, _p2, direction);
       break;
 
     case 9:
-      value = MaterialTensorCalculatorTools::midPrinciple(tensor, direction);
+      value = MaterialTensorCalculatorTools::maxPrinciple(tensor, direction);
```

Comment:
You can have these call the versions with the correctly spelled names now, right?